css/index.html

```diff
@@ -14,8 +14,46 @@
     <main>
 
         <section class="container">
-            <h2 class="title">Buttons</h2>
+            <h2 class="title">Images</h2>
+            <div class="flex-row gap-1">
+                <div>
+                    <img class="img-responsive" src="./assets/nike.jpeg" alt="Nike">
+                </div>
+                <div>
+                    <img class="img-responsive" src="./assets/nike.jpeg" alt="Nike">
+                </div>
+                <div>
+                    <img class="img-responsive" src="./assets/nike.jpeg" alt="Nike">
+                </div>
+                <div>
+                    <img class="img-responsive" src="./assets/nike.jpeg" alt="Nike">
+                </div>
+            </div>
+            <div class="flex-row is-center gap-1">
```

Comment:
What does this class **is-center** do?

Reply:
It's a layout class in flexbox.css needed to center children of flexbox.
It was needed for the container and is not a part of images only.

Reply:
Yeah is-center sounds like a question, the naming convention could have been better